dev-docs/modules/userId.md

```diff
@@ -165,6 +166,38 @@ pbjs.setConfig({
 });
 {% endhighlight %}
 
+### ID+
+
+ID+, a zeotap powered solution, is an open invitation to the entire ecosystem to collaboratively solve the industry challenge of identity resolution in a third-party cookieless environment. 
```

Comment:
Should zeotap be capitalized? It is when first mentioned but isn't in this instance.

Reply:
@MartianTribe it was intentional to use 'z' everywhere for zeotap, also made it a point to use 'Z' only at the start of sentences.